runtime/src/comm/ofi/README.md

```diff
@@ -44,175 +37,257 @@ respect to other operations:
   to the initiator.
 
 - _Message ordering_ settings specify how operations may or may not be
-  reordered with respect to each other by libfabric, the provider, or
-  the network fabric.  Note that "message" in this context is a general
-  term referring to all kinds of communication operations, not just
-  those of the SEND/RECV message interface.
+  reordered with respect to each other.  Note that "message" in this
+  context is a general term referring to all kinds of communication
+  operations, not just those of the SEND/RECV message interface.
 
-The completion level and message ordering settings are specified for the
-entire libfabric domain, that is, the overarching interface to the
-network, rather than individually on each transaction.  (Libfabric does
-allow for some variation in completion level settings on individual
-transactions, but the comm layer does not currently make use of that.)
+A default completion level can be set for the entire libfabric domain,
+that is, the overarching interface to the network, and one can also
+specify completion levels individually for network operations.  The
+message ordering settings can only be specified for the entire domain.
```

Comment:
Isn't it possible to create an endpoint alias with different message orderings?

Reply:
Yeah, good point; I'll adjust this.